src/sdc11073/xml_types/xml_structure.py

```diff
@@ -852,6 +890,11 @@ def __init__(self, sub_element_name: etree_.QName | None, default_py_value: Any
         super().__init__(sub_element_name, ClassCheckConverter(ExtensionLocalValue), default_py_value,
                          is_optional=True)
 
+    def __set__(self, instance: Any, value: Iterable):
```

Comment:
`ExtensionLocalValue` inherits from `list`. I guess `value` should be a `list` then

Reply:
no, tuple or any other iterable works here.